appveyor.yml

```diff
@@ -0,0 +1,61 @@
+image: 
+- Ubuntu1804
+
+environment:
+  major: 4
+  minor: 0
+  release: 0
+  app_version: '$(major).$(minor).$(release).$(APPVEYOR_BUILD_NUMBER)'
+
+version: '$(major).$(minor).$(release).{build}'
+
+dotnet_csproj:
+  patch: true
+  file: '**\*.csproj'
+  assembly_version: '$(app_version)'
+  file_version: '$(app_version)'
+  informational_version: '$(app_version)'
+
+build_script:
+  # linux release build
+  - sh: cmake -BBuild/Release/x64/ -Hsrc/Sundials/ -DCMAKE_BUILD_TYPE=Release -DEXAMPLES_ENABLE_C=OFF -DBUILD_SHARED_LIBS=OFF -DBUILD_STATIC_LIBS=ON -DKLU_ENABLE=OFF
+  - sh: make -C Build/Release/x64/
+  # linux debug build
+#  - sh: cmake -BBuild/Debug/x64/ -Hsrc/Sundials/ -DCMAKE_BUILD_TYPE=Debug
+#  - sh: make -C Build/Debug/x64/
+
+cache:
+  - packages -> **\packages.config  # preserve "packages" directory in the root of build folder but will reset it if packages.config is modified
+
+before_build:
+  - git submodule update --init --recursive
+
+#after_build:
+#  - cmd: nuget pack src\OSPSuite.SimModel\OSPSuite.SimModel.nuspec -version %app_version%
+#  - sh: nuget pack src/OSPSuite.SimModel/OSPSuite.SimModel.Ubuntu18.nuspec -version $app_version
+#  - ps: Get-ChildItem .\OSPSuite.SimModel.*.nupkg | % { Push-AppveyorArtifact $_.FullName -FileName $_.Name }
+
+#test_script:
+#  - ps: dotnet test --no-build --no-restore --logger:Appveyor --configuration:Release --verbosity normal 
+
+branches:
+  only:
+    - master
+    - develop
+
+skip_commits:
+  files:
+    - '*.md'
+
+skip_tags: true
+
+pull_requests: 
+  do_not_increment_build_number: true     
+
+nuget: 
+  disable_publish_on_pr: true  
+
+notifications:
```

Comment:
You don';t need this. This is already in all packages with the global stuff I change last time

Reply:
ok, cool